### PR TITLE
Related content srcset

### PR DESCRIPTION
**Pull request by rich-nguyen**
Ensure that the most relevant container is used for related content, otherwise cards will be configured with incorrect media breakpoints

### CODE REVIEW

File: onward/app/controllers/PopularInTag.scala [hunk 1]
```diff
@@ -23,11 +23,15 @@ object PopularInTag extends Controller with Related with Logging with ExecutionC
     val displayName = Some(dataId)
     val properties = FrontProperties.empty
     val config = CollectionConfigJson.withDefaults(displayName = displayName)
+    val containerDefinition = trails.length match {
+      case 1 => FixedContainers.fixedSmallSlowI
+      case 2 => FixedContainers.fixedSmallSlowII
+      case _ => FixedContainers.fixedMediumFastXII }
```
Comment (robertberry) [inline onward/app/controllers/PopularInTag.scala]: What about 3?

File: onward/app/controllers/RelatedController.scala [hunk 2]
```diff
@@ -35,8 +35,14 @@ object RelatedController extends Controller with Related with Logging with Execu
     val properties = FrontProperties.empty
     val config = CollectionConfigJson.withDefaults(displayName = displayName)
 
-    val container: FaciaContainer = FaciaContainer(1,
-      Fixed(FixedContainers.fixedMediumFastXII),
+    val containerDefinition = trails.length match {
+      case 1 => FixedContainers.fixedSmallSlowI
+      case 2 => FixedContainers.fixedSmallSlowII
+      case _ => FixedContainers.fixedMediumFastXII }
```
Comment (robertberry) [inline onward/app/controllers/RelatedController.scala]: Refactor into a common method?

**Comment by robertberry** [comment]
Good otherwise :+1:

File: onward/app/controllers/PopularInTag.scala (hunk 1 above)
Comment (rich-nguyen) [inline onward/app/controllers/PopularInTag.scala]: a TTT?

Comment (robertberry) [inline onward/app/controllers/PopularInTag.scala]: yup

**Pull request merged**